.github/workflows/test-ios-device.yaml

```diff
@@ -30,6 +30,9 @@ jobs:
           bundler-cache: true
           working-directory: 'packages/patrol/example/ios'
 
+      - name: Set up fastlane
+        run: brew install fastlane
```

Comment:
don't - this will always install the latest version

Install dependencies from the `Gemfile` instead, using [`bundler`](https://bundler.io/).